serverAddons/AAW/addons/AAW_Server/Revive/fn_bloodEffect.sqf

```diff
@@ -69,7 +69,9 @@ private _fadeOut = {
     _tex ctrlCommit _time;
 };
 
-[{GVAR(PPBloodEffectIntensity) = 0}, 1.5, [_texUpper, 1.5]] call CFUNC(wait);
+[{
```

Comment:
i think there should be same acted as with simple if statements that it can only use 1 line to improve readablity